### DISCUSSION TITLE
Windows 11 Taskbar Styler: Crash when using systemtray:NotifyIconView

### DISCUSSION THREAD

**Discussion by Nojevah** [ramensoftware/windhawk-mods#270] [General]
Hello,
First, thanks for Winhawk, it's so powerful, now I have the taskbar I wanted !

I've tried to use Windows 11 Taskbar Styler to reduce space between icons in systray. Thanks to UWPSpy, I managed to find what I wanted (btw, in UWP it's called "SystemTray" but in Windhawk, we need to use "systemtray", without capitals, cost me some time with my limited skills):
The target is systemtray:NotifyIconView
The style is MinWidth=16

When I change this in UWPSpy, it works exactly like what I want, but when I use it in Windhawk, when I click on "restart Explorer", Explorer goes in a crash loop (Werfault) and I need to retire this part to get Explorer starting. It seems to be because of the target more than the style.

Any idea if I can achieve what I want ?

**Comment by Imold** [comment]
I have the same problem trying to set the Width or MinWidth for systemtray:OmniButton#NotificationCenterButton. I was trying to increase the Width so that there would be extra room so everything doesn't get pushed over to make room for the unread notification count when it appears.

**Comment by m417z** [comment]
Should be fixed in Windows 11 Taskbar Styler v1.1.2, please try it.

**Comment by Imold** [comment]
v1.1.2 fixed my issue.

**Comment by Nojevah** [comment]
I confirm the resolution for me too. Thank you very much. Such a perfect taskbar now... But I'm sure at one point I'll want more, thanks Imold for the idea.
